python/raft/dask/common/comms.py

```diff
@@ -90,10 +101,22 @@ def __init__(self, comms_p2p=False, client=None, verbose=False,
                  Dask client to use
         verbose : bool
                   Print verbose logging
+        nccl_root_location : string
+                  Indicates where the NCCL's root node should be located.
+                  ['client', 'worker', 'scheduler' (default)]
+
         """
         self.client = client if client is not None else default_client()
+
         self.comms_p2p = comms_p2p
 
+        if nccl_root_location.lower() not in Comms.valid_nccl_placements:
```

Comment:
Little nitpick- can we call `lower()` once? Maybe assign to `self` first and then raise the error if `self.nccl_root_location not in Comms.valid_nccl_placements`.